src/app/pricing/pricing.component.html

```diff
@@ -0,0 +1 @@
+<p>pricing works!</p>
```

Comment:
pricing component is not part of loginAndRegister feature. Should be commited to different branch.